plugin/src/main/java/io/snyk/languageserver/protocolextension/SnykExtendedLanguageClient.java

```diff
@@ -64,4 +70,19 @@ private void showAuthenticatedMessage() {
     messageParams.setMessage("The authentication token has been stored in Snyk Preferences.");
     ServerMessageHandler.showMessage("Authentication with Snyk successful", messageParams);
   }
+
+  @Override
+  public CompletableFuture<ShowDocumentResult> showDocument(ShowDocumentParams params) {
```

Comment:
Shouldn't we simply update to the latest LSP4E to get this? Or is the release cycle long there?

Reply:
Perhaps worth noting it here, to wipe this code out later.

Reply:
Yeah, we should note it - release cycle is way slower than ours, but I'll add a comment.